New Post: Announcing Alchemy Conf (#385)
subvisual/subvisual.com: src/posts/2020-09-11-announcing-alchemy-conf.md

---
path: /posts/announcing-alchemy-conf/
title: "Announcing Alchemy Conf"

> author: roberto-machado

Reviewer: out of curiosity, why did you feel the need to be a ghost writer, and not put the post under your own name?

Author: Well, it's always been a CEO thing and Roberto's been around for longer and command the conference. He's also been the force to push us over the edge when we were considering moving forward or not. It's only fair.

date: 2020-09-11
tags:
  - development
  - conferences
  - elixir
intro: >
  How important is the community for you? It's been four years since our last
  conference. And now we're back.
---

How important is the community for you? I get asked this sometimes and
inevitably end up reminiscing about Subvisual’s origins. At the time, there
wasn’t a Subvisual, if I’m honest. No, that was still a few years down the road.
There was a Group Buddies, however. A small Ruby and Rails consultancy firm. We
were still taking our first steps and barely made by. This was in 2012, a year
that, in retrospect, defined us. We were presented with the opportunity to
support a small Ruby conference called Ruby Norte and decided to bite the
bullet. Sponsorship was "only” 250€. The use of quotes is not an exaggeration
since, at the time, when we stared at our bank account statement, we saw 200€
staring back at us. The remaining 50€ were put out of our own pockets. I can
never give a correct answer to the question of "how important the community is,”
but **I can look back and proudly say it has always been a part of us.**

A lot more followed. **We started organizing several meetups** (Braga.rb,

> Braga.js, Braga.Design, Braga.Product, Braga BEAM), **helped others by hosting

Reviewer 2: ```suggestion
Braga.js, Braga.Design, Braga.Product, Braga BEAM), helped others by hosting
```

Author: The end of this highlight comes after "Mirror Conf", a few lines below. [Deploy preview here](https://deploy-preview-385--subvisual.netlify.app/blog/posts/announcing-alchemy-conf/). Is it too much though?

them in our office, eventually went around to organize [Ruby Conf
Portugal][rubyconf] from 2014 to 2016. From 2016 to 2018, we also had [Mirror
Conf][mirrorconf]**, a conference that aimed to become a conversation between
designers and front-end developers. Doing this for years on end took its toll,
of course, and we needed a break.

We have evolved from working mainly as a Ruby on Rails consultancy firm to a
venture studio during the past few years, where we help create new startups.
With this transition, we found Elixir to be a valuable alternative that we

> should invest, and we’ve been working with it for more than four years. Many of

Reviewer 2: ```suggestion
should invest in, and we’ve been working with it for more than four years. Many of
```

history-rich, with a fantastic university, filled with young potential. We have
a booming tech ecosystem here, encompassing fully fledged international
companies and promising startups.

**The conference will take place on the 14th and 15th of May 2021. It will be a
single-track conference, with a mixture of guest speakers and CFP selections.**
The first speaker announcements are just right around the corner, and the CFPs
will open soon.

**[The early birds are on sale now][tickets].** We are looking into student
discounts, so if you are a student, send us an email to [hello@alchemyconf.com].

Like the previous conferences we organized, Alchemy Conf is community-driven.
This means it is a non-profit. Our focus is and always has been to inspire the
community. To create an opportunity for developers and enthusiasts to get
together, to share and learn, experience, and network. From the attendees to the
speakers, we want everybody to feel like Alchemy Conf is a little bit of their
own. This is why we want to fully commit to transparency. **We will be making a
transparency report containing the expenses and earnings of the conference. Any

> profit that may arise will be split equally amongst the speakers.**

Reviewer 2: ```suggestion
profit that may arise will be given back to the community.**
```

This gives us more wiggle room, keeping the assurance that it's a not for profit event.

Author: Adding your suggestion, but I'd be more comfortable if we were upfront with what we are doing. I think we should finish the discussion on this topic on Basecamp, before the announcement is published. To make sure everyone is on board.

## The Road to Alchemy Conf

Even though we’re almost seven months away from the start of the conference, we
want everyone to feel like they are a part of it. We want the road to Alchemy

> Conf to be a journey. And **we want to bring closer the community during times of

Reviewer 2: ```suggestion
Conf to be a journey. And **we want to bring the community closer during times of
```

hardship, even though we can’t all be physically together yet.**

We will be setting up **three online workshops in the months leading up to the
conference, all of them around Elixir development, to be given by leading
developers in the community. Each workshop will have a cost of 50€. Taking part
in all three will earn you a free ticket to the conference.** We’re aiming to
encourage and reward taking part in community events. That’s why the total cost

> of the lot, 150€, is the same as an early bird ticket - they won’t be making any

Reviewer 2: ```suggestion
of the lot, 150€, is the same as an early bird ticket - there won't be any
```
